app/views/notes/_note.gpx.builder

```diff
@@ -1,9 +1,14 @@
 xml.wpt("lon" => note.lon, "lat" => note.lat) do
+  xml.time note.created_at.to_s(:iso8601)
+  xml.name "#{t'note.mine.description'} #{note.id}"
```

Comment:
I don't understand the use of `note.mine.description` here? That gives a description of "Description 42" for note 42 which seems very odd...